packages/fields/src/types/Unsplash/Implementation.js

```diff
@@ -14,8 +14,9 @@ const defaultTransforms = {
   fit: 'max',
 };
 
-function transformUserFromApiToKs5(user) {
+function transformUserFromApiToKs5(user, { includeId = false } = {}) {
   return {
+    ...(includeId && { id: user.id }),
     unsplpashId: user.id,
```

Comment:
I know this isn’t changed in this PR but is `unsplpashId` a typo?

Reply:
Hah! It is! I'll fix it in this PR, thanks for catching that 👍